### PR TITLE
[Irina] 2 feature files : sign up & write review. Closes #4

### LINKED ISSUE #4: [Irina] Create feature file with 3 Scenarios

### PR DESCRIPTION
**Pull request by irinazota**

### CODE REVIEW

File: src/test/resources/feature/SignUp.feature [hunk 1]
```diff
@@ -0,0 +1,52 @@
+@Irina
+Feature: SignUp
+
+  Background:
+    Given "Home" is displayed
```
Comment (venache1) [inline src/test/resources/feature/SignUp.feature]: Remove this step as you should handle it in Before Hook

File: src/test/resources/feature/SignUp.feature [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    And user clicks on "Sign in"
+    And "Login" is displayed
+    And user clicks on "Email address field"
+    And user enters "lalalala@emailll.com"
+    And user clicks on "Create an account"
+    And "CREATE AN ACCOUNT" is displayed
```
Comment (venache1) [inline src/test/resources/feature/SignUp.feature]: You can replace this with an single step like **user is on Create an account page** and handle all the logic in StepDef

File: src/test/resources/feature/SignUp.feature [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+  Scenario: SignUp - successful
+    When user enters VALID data into mandatory fields:
```
Comment (venache1) [inline src/test/resources/feature/SignUp.feature]: Why is necesarry to specify that data is VALID and fields are mandatory? You can specify this in Scenario title and make this step more generic one

File: src/test/resources/feature/SignUp.feature [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+      | Zip/Postal Code                               | 00000                    |
+      | Country                                       | United States            |
+      | Mobile phone                                  | 105454545                |
+      | Assign an address alias for future reference. | Home address             |
+    And user clicks on "Register"
+    Then "My Account" is displayed
+    And user name is displayed in Navigation bar
+
+  Scenario: SignUp - unsuccessful
+    When user enters INVALID data into mandatory fields:
```
Comment (venache1) [inline src/test/resources/feature/SignUp.feature]: See the comment above, please

File: src/test/resources/feature/SignUp.feature [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+      | Address                                       |             |
+      | City                                          |             |
+      | State                                         |             |
+      | Zip/Postal Code                               | A0000       |
+      | Country                                       |             |
+      | Mobile phone                                  |             |
+      | Assign an address alias for future reference. |             |
+    And user clicks on "Register"
+    Then "CREATE AN ACCOUNT" is displayed
+    And errors are displayed
```
Comment (venache1) [inline src/test/resources/feature/SignUp.feature]: Here will be nice to specify which errors are displayed

File: src/test/resources/feature/SignUp.feature (hunk 4 above)
Comment (irinazota) [reply src/test/resources/feature/SignUp.feature]: Done

File: src/test/resources/feature/SignUp.feature (hunk 5 above)
Comment (irinazota) [reply src/test/resources/feature/SignUp.feature]: Done

File: src/test/resources/feature/SignUp.feature (hunk 3 above)
Comment (irinazota) [reply src/test/resources/feature/SignUp.feature]: Done

File: src/test/resources/feature/SignUp.feature (hunk 2 above)
Comment (irinazota) [reply src/test/resources/feature/SignUp.feature]: Done

File: src/test/resources/feature/SignUp.feature (hunk 1 above)
Comment (irinazota) [reply src/test/resources/feature/SignUp.feature]: Done

File: src/test/resources/feature/SignUp.feature [hunk 6]
```diff
@@ -0,0 +1,54 @@
+@Irina
+Feature: SignUp
+
+  Background:
+    Given user is on "CREATE AN ACCOUNT" page
+
+  Scenario: SignUp with valid data - successful
```
Comment (venache1) [inline src/test/resources/feature/SignUp.feature]: Would be possible to SignUp with **valid** data **unsuccessfully**?

File: src/test/resources/feature/SignUp.feature [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+      | State                                         | Alabama                  |
+      | Zip/Postal Code                               | 00000                    |
+      | Country                                       | United States            |
+      | Mobile phone                                  | 105454545                |
+      | Assign an address alias for future reference. | Home address             |
+    And user clicks on "Register"
+    Then "My Account" is displayed
+    And user name is displayed in Navigation bar
+
+  Scenario: SignUp with invalid data - unsuccessful
```
Comment (venache1) [inline src/test/resources/feature/SignUp.feature]: Same here

File: src/test/resources/feature/WriteReview.feature [hunk 8]
```diff
@@ -0,0 +1,17 @@
+@Irina
+Feature:   Write Review
+
+  Scenario: WriteReview - Positive
+    Given user is logged in
+    And "Home" is displayed
```
Comment (venache1) [inline src/test/resources/feature/WriteReview.feature]: We assume that after logging in Home page should be dispayed. We can assert that in first step

File: src/test/resources/feature/WriteReview.feature [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+    And user clicks on "item1"
+    And "item1's detailed page" is displayed
```
Comment (venache1) [inline src/test/resources/feature/WriteReview.feature]: Same here, we can assert that this page is displayed in previous step

File: src/test/resources/feature/WriteReview.feature [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
+    And user clicks on "Write a review"
+    And "WRITE A REVIEW pop-up" is displayed
```
Comment (venache1) [inline src/test/resources/feature/WriteReview.feature]: Same here

File: src/test/resources/feature/WriteReview.feature [hunk 11]
```diff
@@ -0,0 +1,17 @@
...
+    And user clicks on "Quality 3 stars"
+    And user clicks on "Title field"
+    And user enters "VALID DATA"
```
Comment (venache1) [inline src/test/resources/feature/WriteReview.feature]: Can we have invalid data for Title field?

File: src/test/resources/feature/WriteReview.feature [hunk 12]
```diff
@@ -0,0 +1,17 @@
...
+    And user clicks on "Comment field"
+    And user enters "VALID DATA"
```
Comment (venache1) [inline src/test/resources/feature/WriteReview.feature]: Can we have invalid data for Comment field? I sugest to combine those 2 steps and to have something like:
**user enters "Test comment" in Comment field**
You can do the same for Title field

Comment (irinazota) [reply src/test/resources/feature/WriteReview.feature]: Done

File: src/test/resources/feature/WriteReview.feature (hunk 11 above)
Comment (irinazota) [reply src/test/resources/feature/WriteReview.feature]: That would be blank value, for ex.

File: src/test/resources/feature/WriteReview.feature (hunk 10 above)
Comment (irinazota) [reply src/test/resources/feature/WriteReview.feature]: Done

File: src/test/resources/feature/WriteReview.feature (hunk 9 above)
Comment (irinazota) [reply src/test/resources/feature/WriteReview.feature]: Done

File: src/test/resources/feature/WriteReview.feature (hunk 8 above)
Comment (irinazota) [reply src/test/resources/feature/WriteReview.feature]: 'My Account' screen is displayed after signing in. The purpose of this step is that user 'navigates' to Home. Renamed the step

File: src/test/resources/feature/SignUp.feature (hunk 7 above)
Comment (irinazota) [reply src/test/resources/feature/SignUp.feature]: Done

File: src/test/resources/feature/SignUp.feature (hunk 6 above)
Comment (irinazota) [reply src/test/resources/feature/SignUp.feature]: Done

**Review by venache1 — APPROVED** [review]

**Pull request merged**